internal/dataplane/parser/translate_ingress_test.go

```diff
@@ -164,3 +165,110 @@ func TestGetDefaultBackendService(t *testing.T) {
 		})
 	}
 }
+
+func TestRewriteURIAnnotation(t *testing.T) {
+	pathTypePrefix := netv1.PathTypePrefix
+	var nilKongPlugins []kong.Plugin
+
+	someIngress := func(name, rewriteURI string) netv1.Ingress {
+		return netv1.Ingress{
+			TypeMeta: metav1.TypeMeta{Kind: "Ingress"},
```

Comment:
```suggestion
```
it's redundant imo

Reply:
Without this line, the GVK of the ingress is empty, which will cause`PushResourceFailure` failed while we need check the `ResourceFailure`.